app/controllers/account/pages_controller.rb

```diff
@@ -33,4 +33,12 @@ def resource
   def collection
     parent.pages
   end
+
+  def list_without_params
+    collection.by_rating.page(params[:page]).per(10)
+  end
+
+  def list_with_params
+    Page.__elasticsearch__.search(params[:q]).page(params[:page]).per(10)
```

Comment:
I can't make search in scope of 'current_user.companies' (not all pages in db).
I've put 'collection' instead of 'Page' here, but it doesn't work too.